src/controllers/cohort.js

```diff
@@ -51,3 +52,37 @@ export const getCohortById = async (req, res) => {
     return sendMessageResponse(res, 500, 'Unable to get cohort')
   }
 }
+
+export const getAllCohortExercises = async (req, res) => {
+  const cohortId = Number(req.params.id)
+  const whereData = {}
+  whereData.cohortId = cohortId
+
+  try {
+    const foundExercise = await CohortExercise.findAll({ whereData })
+
+    const formattedExercise = foundExercise.map((exercise) => {
+      return {
+        ...exercise.toJSON().exercise
+      }
+    })
+    return sendDataResponse(res, 200, { cohortExercises: formattedExercise })
+  } catch (e) {
+    return sendMessageResponse(res, 500, 'Unable to get exercise')
+  }
+}
+
+export const createCohortExercise = async (req, res) => {
+  const { exerciseId: exerciseID, cohortId: cohortID } = req.body
```

Comment:
What's the reasoning to map the keys to different variables? Can you not work with `exerciseId` and `cohortId` ?

Reply:
exerciseId: exerciseId was not needed, so I have left it as {exerciseId, cohortId} = req.body